src/honey_badger.rs

```diff
@@ -9,6 +9,40 @@ use serde::Serialize;
 use common_subset::{self, CommonSubset};
 use messaging::TargetedMessage;
 
+/// A Honey Badger error.
```

Comment:
Likewise, I don't see why these definitions should be moved.

Reply:
See above comment for my reasoning on putting errors at the top of the file, especially in circumstances like the following:

```
use some_crate;

pub enum Error {
    SomeCrate(some_crate::Error)
}
```
I think that it is another (loosely held) convention to place types that are composed of other types closer to each other in the file as it aids readability. For example:

```
struct A;
struct B;
type MyTuple = (A, B);
```

is far more readable than something like:

```
struct A;
struct B;
... 100 lines of code ...
type MyTuple = (A, B);
```

Reply:
I don't feel very strongly about this change, if you have a preference I am happy to change things around.